### DISCUSSION TITLE
Domain name Instead ip adress

### DISCUSSION THREAD

**Discussion by TheKogoro** [thijsvanloef/palworld-server-docker#544] [Q&A] [answered]
Hello, i can connect sucefully with my domain name Instead my ip but, If i check my Server on the website https://www.battlemetrics.com/servers/palworld/ 
On "adress" i get only my ip, no domain name. I See somes other Server have domain name Instead ip in "Adresse". I have tried to put my domain name in the "public ip" settings but it didnt change anything. I dit something wrong or the settings that permit  that doesnt existe actually on this docker Server ?

**Comment by Diyagi** [comment, marked as answer]
This is either something you can do by doing the battlemetrics setup or battlemetrics is resolving reverse domain names (PTR), it has nothing to do with how you configure the server.

Edit: i did an quick check and it seems that u can achieve that by going thru the setup in battlemetrics.
